README.md

```diff
@@ -27,14 +33,19 @@ $ cat <<EOF kubectl -f -
 apiVersion: v1
 kind: Pod
 metadata:
-  name: hl-smi
+  name: habana-ai-demo
   namespace: habana-ai-operator
 spec:
   restartPolicy: OnFailure
   containers:
-  - image: ghcr.io/fabiendupont/habana-ai-hl-smi:latest
+  - image: vault.habana.ai/gaudi-docker/1.6.1/ubuntu20.04/habanalabs/tensorflow-installer-tf-cpu-2.9.1:latest
     imagePullPolicy: IfNotPresent
-    name: hl-smi
+    name: habana-ai-base-container
+    workindDir: /root
+    command:
+      - "echo"
+    args:
+      - "Hello world!"
```

Comment:
I agree and I have updated the command to list all Habana devices, i.e. `hl-smi -L`, since I think that there is no need for this sample workload to be any more real or complex.

The container image was changed to one hosted in the habana.ai vault, as the previous container image is not currently available.

What do you think?

> We can discuss later on about whether we should create a smaller container image or even one with real workload.

Reply:
If that image has all it needs, it's indeed better.

Reply:
The image has everything we need indeed :+1: